### DISCUSSION TITLE
Group input components?

### DISCUSSION THREAD

**Discussion by cmple** [quasarframework/quasar#12587] [General - Components / Directives / etc]
Hi,

What is the best way to group an input[text] component?

For example, displaying a date range or a number range input fields with a text separator under a single stacked label:
[q-input['date_from'], ' -> ', q-input['date_to']]

Thank you!

**Comment by MilosPaunovic** [comment]
[https://codepen.io/milospaunovic/pen/xxPamEP?editors=1011](https://codepen.io/milospaunovic/pen/xxPamEP?editors=1011)
